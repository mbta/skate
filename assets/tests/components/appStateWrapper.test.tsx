import React from "react"
import renderer from "react-test-renderer"
import AppStateWrapper from "../../src/components/appStateWrapper"

test("renders", () => {
  const tree = renderer.create(<AppStateWrapper />).toJSON()
  expect(tree).toMatchSnapshot()
})
