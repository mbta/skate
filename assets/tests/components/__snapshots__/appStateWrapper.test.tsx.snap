// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="m-app"
>
  <div
    className="m-route-picker visible"
  >
    <div
      className="m-route-picker__tab"
    >
      <button
        className="m-route-picker__tab-button"
        onClick={[Function]}
      >
        <span
          className=""
          dangerouslySetInnerHTML={
            Object {
              "__html": "SVG",
            }
          }
        />
      </button>
    </div>
    <ul
      className="m-route-picker__selected-routes"
    />
    <div
      className="m-route-filter"
    >
      <select
        className="m-route-filter__type"
        onChange={[Function]}
        value="id"
      >
        <option
          value="id"
        >
          Route ID
        </option>
      </select>
      <div
        className="m-route-filter__text"
      >
        <input
          className="m-route-filter__input"
          onChange={[Function]}
          onKeyDown={[Function]}
          type="text"
          value=""
        />
        <button
          className="m-route-filter__clear"
          onClick={[Function]}
        >
          <span
            className=""
            dangerouslySetInnerHTML={
              Object {
                "__html": "SVG",
              }
            }
          />
        </button>
      </div>
    </div>
    loading...
  </div>
  <div
    className="m-route-ladders"
  />
</div>
`;
